test/libs/datasets/statistical_areas_test.py

```diff
@@ -60,3 +64,29 @@ def test_aggregate():
         pd.to_datetime("2020-05-03"): 6,
         pd.to_datetime("2020-05-04"): 4,
     }
+
+
+@pytest.mark.parametrize("reporting_ratio,expected_na", [(0.8, False), (0.91, True)])
+def test_aggregate_reporting_ratio(reporting_ratio, expected_na):
+    ny_region = Region.from_state("NY")
+    az_region = Region.from_state("AZ")
+    us_region = Region.from_iso1("us")
```

Comment:
unless there is good reason to use state -> US aggregation please use region types expected by CountyToCBSAAggregator.

Reply:
The best reason is for creating the test dataset, which expects regions

Reply:
It is really just as easy to make county and CBSA regions: https://github.com/covid-projections/covid-data-model/blob/main/libs/pipeline.py#L108-L132